### DISCUSSION TITLE
A-Tap work only after releasing keys

### DISCUSSION THREAD

**Discussion by moilong** [labwc/labwc#1110] [General]
I think it should change nextwindow on pressing alt+tab but window change occurs after releasing keys. labwc version 0.6.4, installed from xbps.

**Comment by Consolatis** [comment]
By default the window-switcher preview is enabled which means that labwc should show the target window on press: `<windowSwitcher preview="yes" />`. Is that not the case?

**Reply by moilong** [reply]
My mistake. I set that to 'no' thinking to turn off windowswitcher popup. Thankyou and sorry!

**Comment by johanmalm** [comment]
> I think it should change nextwindow on pressing alt+tab but window change occurs after releasing keys. labwc version 0.6.4, installed from xbps.

We currently switch window on press, but remain in 'cycle-mode' (i.e. window-switcher showing) until release. This is consistent with openbox/mutter/windows/etc and I personally expect it to work.

There isn't something weird going on with your keybinds is there? We've just fixed a bug in how we handle keybinds, so might be worth building master and trying.

Will close for now, but please say if we're missing a valid use-case here.
